pom.xml

```diff
@@ -60,9 +60,6 @@
 		<!-- by default we run perun in memory -->
 		<spring.profiles.default>default</spring.profiles.default>
 
-		<!-- redefine versions of some libraries defined by Spring Boot Starter Parent -->
-		<tomcat.version>8.5.57</tomcat.version>
```

Comment:
Can you please move this change outside of this pull-request? I will prepare change for this including update of cargo plugin. Its not really related to java update.

Reply:
removed